spec/version.dd

```diff
@@ -352,6 +352,8 @@ $(H3 $(LEGACY_LNAME2 PredefinedVersions, predefined-versions, Predefined Version
         $(TROW $(ARGS $(D unittest)) , $(ARGS $(DDLINK spec/unittest, Unit Tests, Unit tests) are enabled
                 (command line switch $(DDSUBLINK dmd, switch-unittest, $(TT -unittest)))))
         $(TROW $(ARGS $(D assert)) , $(ARGS Checks are being emitted for $(GLINK2 expression, AssertExpression)s))
+        $(TROW $(ARGS $(D D_PreConditions)) , $(ARGS Checks are being emitted for $(GLINK2 expression, InContractExpression)s))
```

Comment:
`InContractExpression` links to the expression page, but it's defined in the function page.